shared/profile/post-proof/container.js

```diff
@@ -42,8 +42,7 @@ const mapDispatchToProps = dispatch => ({
   copyToClipboard: text => dispatch(ConfigGen.createCopyToClipboard({text})),
 })
 
-export default compose(
-  // FlowIssue on 86
+export default (compose(
```

Comment:
some of these exports are becoming `any`, do we have a workaround for these cases?

Reply:
not now